doc/Requirements Specification/Parts/Non-Functional Requirements.lyx

```diff
@@ -430,5 +430,274 @@ gls{SoMoX}
 .
 \end_layout
 
+\begin_layout Subsection
+GUI model
+\end_layout
+
+\begin_layout Standard
+The user has multiple options to launch the analysis:
+\end_layout
+
+\begin_layout Enumerate
+To analyse the whole project, there is an entry 
+\begin_inset Quotes eld
+\end_inset
+
+Analyse with Beagle
+\begin_inset Quotes erd
+\end_inset
+
+ in the context menu of the 
+\begin_inset ERT
+status open
+
+\begin_layout Plain Layout
+
+
+\backslash
+lstinline{.repository}
+\end_layout
+
+\end_inset
+
+ or 
+\begin_inset ERT
+status open
+
+\begin_layout Plain Layout
+
+
+\backslash
+lstinline{.repository_diagram}
+\end_layout
+
+\end_inset
+
+ file in eclipse.
```

Comment:
Yesterday, we discussed that these entries are in the project’s context menu and maybe in the global “Project” menu entry.